lighty-examples/lighty-gnmi-community-restconf-app/README.md

```diff
@@ -0,0 +1,81 @@
+# lighty.io gNMI/RESTCONF application
+This application provides gNMI south-bound utilized with RESTCONF interface to manage gNMI devices on the network.
+Application works as standalone SDN controller. It is capable to connect to gNMI devices and expose connected devices
+over RESTCONF north-bound APIs. In this application will be started gNMI simulator as gNMI target and all operation will
```

Comment:
In this application gNMI simulator will be started as gNMI target and all operations will..